relay-server/src/integrations/mod.rs

```diff
@@ -93,3 +99,11 @@ pub enum OtelFormat {
     /// OTeL data in a JSON container.
     Json,
 }
+
+#[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
+pub enum VercelLogDrainFormat {
+    // Vercel Log Drain data in a json array payload
+    Json,
+    // Vercel Log Drain data in a newline delimited json payload
+    NDJson,
```

Comment:
```suggestion
    NdJson,
```

Even if it looks wrong, according to Rust code this is the proper capitalization.